docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -392,6 +392,44 @@ The configuration items within a configuration category can each be defined as o
    * - object
      - A complex configuration type with multiple elements that may be used within *list* and *kvlist* items only, it is not possible to have *object* type items outside of a list. Object type configuration items have a set of *properties* defined, each of which is itself a configuration item. 
 
+List
+####
+
+A simple list would be defined using the JSON shown below
+
+.. code-block:: JSON
+
+ "tags" : {
+                "description" : "A set of tag names on which to operate",
+                "type" : "list",
+                "items" : "string",
+                "default" : "[ \"speed\", \"temperature\", \"voltage\" ]",
+                "order" : "4",
+                "displayName" : "Labels"
+           }
+
+The returned value for such a configuration item is merely an array of strings.
+
+.. code-block:: JSON
+
+    [ "speed", "temperature", "voltage", "current", "status" ]
```

Comment:
`[ \"speed\", \"temperature\", \"voltage\" ]`

Reply:
This is the returned value, not shown as a string, so no need for the escaping

Reply:
But `"current", "status"` are extra values as per given JSON example. I believe we should match up the JSON examples with result otherwise user may confuse.